pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -37,6 +38,10 @@ python3.pkgs.buildPythonApplication rec {
     hidapi=${ hidapi }/lib/
```

Comment:
Can you please move the comment above `postPatch` into this block, so it's at the same level as your new comment?

Reply:
Done

Reply:
Ahh no, I meant the other way. 🙃 

Leave your comment where it is and move my comment inside the block:


```nix
postPatch = ''
  # handheld-daemon contains a fork of the python module `hid`, so this hook
  # is borrowed from the `hid` derivation.
  hidapi=${ hidapi }/lib/
  test -d $hidapi || { echo "ERROR: $hidapi doesn't exist, please update/fix this build expression."; exit 1; }
  sed -i -e "s|libhidapi|$hidapi/libhidapi|" src/hhd/controller/lib/hid.py

  # The generated udev rules point to /bin/chroot, which does not exist in NixOS
  chmod=${ toybox }/bin/chmod
  sed -i -e "s|/bin/chmod|$chmod|" src/hhd/controller/lib/hide.py
'';
```

Reply:
Did this change get pushed.  I'm still seeing both comments smooshed above the hook.

Reply:
Oh no, I completely overlooked that, will fix now

Reply:
Ok it's pushed now